include/wx/aui/auibook.h

```diff
@@ -356,6 +356,8 @@ class WXDLLIMPEXP_AUI wxAuiNotebook : public wxNavigationEnabled<wxBookCtrlBase>
     virtual bool InsertPage(size_t index, wxWindow *page, const wxString &text,
                             bool select, int imageId) wxOVERRIDE;
 
+    wxSize DoGetBestSize() const;
```

Comment:
This should use `wxOVERRIDE` (not only for clarity but also because without it we're going to get clang warnings about inconsistent use of override).

Reply:
Done. I also added `virtual` as all other `DoGetBestSize` methods use it.